website/docs/components/app-header/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,23 @@
+## Conformance rating
+
+<Doc::Badge @type="success">Conformant</Doc::Badge>
+
+When used as recommended, there should not be any WCAG conformance issues with this component.
+
+## Focus
```

Comment:
@MelSumner I've moved some of the content from the `SideNav` docs here around focus order because it seemed to make sense, can you take a look at this, as well as the success criteria?